### DISCUSSION TITLE
Store lyrics in MediaFile

### DISCUSSION THREAD

**Discussion by masterT** [beetbox/beets#4070] [Ideas]
I don't know if this is possible, but it would be nice to be able to store lyrics (from the _lyrics_ plugin) inside the file (_MediaFile_).  I know the _MediaFile_ support lyrics (https://mediafile.readthedocs.io/en/latest/), is there a reason why the lyrics are not stored in the file? Is it this is possible?

**Comment by sampsyo** [comment]
Hmm; I thought they were! Maybe you just need a `beet write` to sync up the metadata with the database for the files you're looking at? You can try `beet info` (after enabling the `info` plugin) to check out the metadata that's in files.

**Reply by masterT** [reply]
You're right, I tried to import my songs into a self-hosted streaming application and it did not detect the lyrics. I checked the code of the lyrics plugin and did not find anything related to _MediaFile_. I'm not a Python programmer and I don't know the whole _Beets_ ecosystem, I should check the file output first. 

Thank you and sorry for your time. 😅
